controller/registry/private.py

```diff
@@ -87,8 +87,7 @@ def _put_first_image(repository_path):
 
 
 def _api_call(endpoint, data=None, headers={}, cookies=None, request_type='GET'):
-    # FIXME: update API calls for docker 0.10.0+
-    base_headers = {'user-agent': 'docker/0.9.0'}
+    base_headers = {'user-agent': 'docker/0.10.1'}
```

Comment:
I think that would [act the same](https://github.com/docker/docker-registry/blob/13c1fcbd4ae26bd8d0bb3f9116a6253f4db67014/docker_registry/images.py#L200) as "docker/0.10.1" in my reading of the code. I took this user-agent string from [docker-registry's tests](https://github.com/docker/docker-registry/blob/13c1fcbd4ae26bd8d0bb3f9116a6253f4db67014/tests/base.py#L28).

Is there some reason to change the UA string other than cosmetic?